lib/src/binary_codec.dart

```diff
@@ -199,6 +199,16 @@ class PostgresBinaryEncoder extends Converter<dynamic, Uint8List> {
           }
           return outBuffer;
         }
+      case PostgreSQLDataType.geometry:
+        {
+          if (value is Geometry) {
+            return castBytes(
+              utf8.encode(
+                value.toText(),
+              ),
+            ); //TODO: Convert to Uint8List. Need help here. Using toText() in raw sql inserts it well.
```

Comment:
Please help me to understand this TODO: it seems that it is already implemented. Is there something missing?

Reply:
Actually, this should have been added to the PostgresTextEncoder. I've added it in the last commit

Reply:
b0c853a adds conversion of geometry objects from dart to string using toText() which returns the text in which postgis can understand